packages/binding-file/package.json

```diff
@@ -14,21 +14,7 @@
     "main": "dist/file.js",
     "types": "dist/file.d.ts",
     "devDependencies": {
-        "@node-wot/td-tools": "0.8.9",
-        "@types/node": "16.18.35",
-        "@typescript-eslint/eslint-plugin": "^4.30.0",
-        "@typescript-eslint/parser": "^4.30.0",
-        "eslint": "^7.32.0",
-        "eslint-config-prettier": "^8.3.0",
-        "eslint-config-standard": "^16.0.3",
-        "eslint-plugin-import": "^2.24.2",
-        "eslint-plugin-node": "^11.1.0",
-        "eslint-plugin-promise": "^5.1.0",
-        "eslint-plugin-unused-imports": "^1.1.4",
-        "prettier": "^2.3.2",
-        "typescript": "4.7.4",
-        "typescript-standard": "^0.3.36",
-        "wot-typescript-definitions": "0.8.0-SNAPSHOT.26"
+        "@node-wot/td-tools": "0.8.9"
```

Comment:
I am surprised to see still `td-tools` ...

Reply:
Hmm, yeah, I think in the other packages they are used as a dependency and not as a devDependency, I think in this case it works as `td-tools` definitions are only used for typing. Or should `td-tools` generally be replaced with `wot-typescript-definitions`?

Reply:
I see, let's keep it as is (or as it was)